README.md

```diff
@@ -1,29 +1,60 @@
-# Math-Magician
+# 📖 [Bookstore] <a name="Bookstore: connect to API"></a>
 
-This is a book store app build by react library.
+> n this project, I connected my React-Redux application to an existing API to create and remove books in a remote server.
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your projection so it says/summarizes the whole project and not only the milestone you did. A description like the one you add here is for the pull request. 
But the project description should describe the whole application.